### ISSUE TITLE
Group translations in subfolder

### ISSUE THREAD

**Issue by tooomm** [AutoDarkMode/Windows-Auto-Night-Mode#301]
**Describe the enhancement or feature you'd like**
Instead having many many folders used for translations on the root level, gather them in a single `translations` or `localisation` folder with subfolders for each language.

**Additional context**
I can not even display half of the folders on my external monitor and have to scroll a lot to find the executable to start the app.
Some tidying and arranging would be great there.

I'm using the zip file in case the folder structure looks different when the app is installed.

**Comment by Spiritreader** [comment]
currently not possible without hacky workaround, which we'd like to avoid. See: https://github.com/dotnet/sdk/issues/10366

**Comment by tooomm** [comment]
>I recommend creating shortcuts 😉

I try to avoid workarounds as well and prefer a cleaner app folder in general. ;)

Thanks for linking the upstream issue. Maybe there will be an improved way of handling this at one point.

**Comment by Jay-o-Way** [comment]
Is this still an issue?

**Comment by tooomm** [comment]
In version 10.4.2.29, yes.

The `adm-app` folder has still 363 items of which 88 are folders (mostly for translation files with others mixed in between).
This applies to both, the installation and the portable version.

But in particular for the portable/zip use case, it's relevant as one would need to go to the folder and search the right exe file to start and open the tool between hundreds of files.
For installed apps, that rarely happens. Nonetheless it's messy there, too.

**Comment by ChenYiLins** [comment]
This problem is still an unlikely problem to be solved directly for a long time.

But we can use a flexible way. The current program has two folders, adm-app and adm-updater. We can add an exe under the main folder, and at the same time we can do version control. This should be more friendly to users who use portable.

**Comment by tooomm** [comment]
>The current program has two folders, adm-app and adm-updater. We can add an exe under the main folder, and at the same time we can do version control. This should be more friendly to users who use portable.

Yes, having a new exe file in the top level folder, would be very convenient for portable app users!
![Image](https://github.com/user-attachments/assets/0fb0c009-c4dc-4156-af70-5dd17f1bee36)

**Comment by Spiritreader** [comment]
> This problem is still an unlikely problem to be solved directly for a long time.
> 
> But we can use a flexible way. The current program has two folders, adm-app and adm-updater. We can add an exe under the main folder, and at the same time we can do version control. This should be more friendly to users who use portable.

That is not possible. The updater supports no such thing. It expects a package which is a self contained folder. This is necessary to streamline the update process and make it safe

**Comment by tooomm** [comment]
With the new v11 Beta, all translations next to many other files are still all dumped in the `adm-app/ui` folder and it's annoying for portable users to find the actual `AutoDarkModeApp.exe` to run the tool:
<img width="790" height="984" alt="Image" src="https://github.com/user-attachments/assets/4d948869-ef98-406e-bb15-3c490e67d41d" />

You noted that a new exe on level of the adm-updater an adm-app folders is not possible.
Is gathering all translations files in a `Translations` subfolder possible? When also putting libraries in e.g. a `dll` subfolder, the folder content looks way cleaner and the exe file is not hidden in a sea of files anymore (following the concept used for `Assets` folder already):
<img width="853" height="983" alt="Image" src="https://github.com/user-attachments/assets/09909ad9-586a-48c7-b847-b4fcc7ed666e" />

**Comment by Spiritreader** [comment]
> With the new v11 Beta, all translations next to many other files are still all dumped in the `adm-app/ui` folder and it's annoying for portable users to find the actual `AutoDarkModeApp.exe` to run the tool: <img alt="Image" width="790" height="984" src="https://private-user-images.githubusercontent.com/9874850/473697799-4d948869-ef98-406e-bb15-3c490e67d41d.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTU5OTYzOTcsIm5iZiI6MTc1NTk5NjA5NywicGF0aCI6Ii85ODc0ODUwLzQ3MzY5Nzc5OS00ZDk0ODg2OS1lZjk4LTQwNmUtYmIxNS0zYzQ5MGU2N2Q0MWQucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI1MDgyNCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNTA4MjRUMDA0MTM3WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ODA1NTIwN2UxNTIyMDdjNDcwZjFjZTlhYTNjYzhhMzRhZjIxMzc1NjU1YTI1ZDdhMjEyOTQzMzMwOTgzOTNkNCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.8PsG6ohukFBIkn_6-hEWNJb-nDMnM0LYSA_nE8E-ojM">
> 
> You noted that a new exe on level of the adm-updater an adm-app folders is not possible. Is gathering all translations files in a `Translations` subfolder possible? When also putting libraries in e.g. a `dll` subfolder, the folder content looks way cleaner and the exe file is not hidden in a sea of files anymore (following the concept used for `Assets` folder already): <img alt="Image" width="853" height="983" src="https://private-user-images.githubusercontent.com/9874850/473698068-09909ad9-586a-48c7-b847-b4fcc7ed666e.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTU5OTYzOTcsIm5iZiI6MTc1NTk5NjA5NywicGF0aCI6Ii85ODc0ODUwLzQ3MzY5ODA2OC0wOTkwOWFkOS01ODZhLTQ4YzctYjg0Ny1iNGZjYzdlZDY2NmUucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI1MDgyNCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNTA4MjRUMDA0MTM3WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZGQxOTBlOGM5OTkwZjgxZDBhYjZkODJkMTVjNDYyMzEyMzg0ZGI1ZDRmMDEzNjA5ZjdiZGMyZGZmY2E3ODJlYyZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.Dvu3MToFSP6ftrc03rI7nxmnbnsvqvOzVKUjm7dBeww">

No it is not possible. The reason it's structured like this now is because the dotnet runtimes slightly differ between the backend and frontend ever since the inception of the winui3 build using WinAppSDK produces different DLLs that are no longer shareable.
This packaging is required to make the applications launch.
The feature to change linking location for dlls was removed from dotnet and very likely won't come back. I'd suggest to take it up in the previously linked issue and contact someone there.

What would potentially be possible is to create a lightweight launcher executable, but all of this doesn't really warrant the effort or maintenance really that this would create.
